### PR TITLE
Create header and headline sections

### PR DESCRIPTION
**Pull request by adebsa2401**
We created the header and the headline section of the mobile version of the [template 2](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).
- A section tag is used for the headline section instead of div
- An a tag has been used for the logo instead of label tag
- The menu hamburger icon has been wrapped into a nav tag
- background-image attribute is used to set the background image
- The css code has been refactored

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,57 @@
...
+                <svg class="icon" xmlns="http://www.w3.org/2000/svg" fill="#FFFFFF" class="bi bi-medium" viewBox="0 0 16 16">
+                    <path d="M9.025 8c0 2.485-2.02 4.5-4.513 4.5A4.506 4.506 0 0 1 0 8c0-2.486 2.02-4.5 4.512-4.5A4.506 4.506 0 0 1 9.025 8zm4.95 0c0 2.34-1.01 4.236-2.256 4.236-1.246 0-2.256-1.897-2.256-4.236 0-2.34 1.01-4.236 2.256-4.236 1.246 0 2.256 1.897 2.256 4.236zM16 8c0 2.096-.355 3.795-.794 3.795-.438 0-.793-1.7-.793-3.795 0-2.096.355-3.795.794-3.795.438 0 .793 1.699.793 3.795z"/>
+                </svg>
+            </div>
+            <!-- social medias end -->
+            
+            <!-- background image start -->
+            <img id="backgroup-image" src="assets/background-image.png" alt=""/>
+            <!-- background image end -->
+        </div>
```
Comment (Alfredbis29) [inline index.html:54]: - [ ] Looks great to have the background Image for your portfolio :+1:  but the great idea to have this correctly done it's by using the `background-image` CSS property for it. Please let's change this code here

File: index.html [hunk 2]
```diff
@@ -1,13 +1,57 @@
...
+        <div id="headline">
+            <h1>Hey There. I'm Ben Salès</h1>
+            <h2>I am a Software Developer</h2>
+            <p>
+                I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+
+            <!-- social medias start -->
+            <div id="social-medias">
```
Comment (Alfredbis29) [inline index.html:32]: - [ ] Kindly use the `section` tag instead of the `div` here.

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
Hi @adebsa2401 , 

Good job so far :100: !


### Highlighted
- Good readMe :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- No linter errors :heavy_check_mark: 
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

- [x] Very good design so far, you did a great job here but we can improve something to make it attractive to recruiters.
   - Here you can see how the width of the page is fixed but it should have `width:100%` to re-flow according to the screen width.
 
mobilescreen


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 3]
```diff
@@ -1,13 +1,57 @@
...
         <link rel="stylesheet" href="style.css">
     </head>
 
     <body>
-        
+        <!-- header start -->
+        <header>
+            <label id="author-name">
+                Adebsa
+            </label>
```
[the remark this answers is not in the document]
Comment (mgmediaweb) [reply index.html]: Good job following the advice of the previous reviewer

File: index.html (hunk 1 above)
Comment (mgmediaweb) [reply index.html]: Good job following the advice of the previous reviewer

File: index.html (hunk 2 above)
Comment (mgmediaweb) [reply index.html]: Good job following the advice of the previous reviewer

**Pull request merged**